### PR TITLE
release(v0.11.1): prepare release

### PR DESCRIPTION
**Pull request by smira**
This is the official v0.11.1 release.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

**Review by rsmitty — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,513 @@
...
+* containerd was updated to 1.5.2
+* Linux kernel was updated to 5.10.45
+* Kubernetes was updated to 1.21.2
+* etcd was updated to 3.4.16
+* CoreDNS was updated to 1.8.4
+
+
+### CoreDNS
+
+Added the flag `cluster.coreDNS.disabled` to coreDNS deployment during the cluster bootstrap.
```
Comment (Unix4ever) [inline CHANGELOG.md:32]: Found my typo here :facepalm: 
```suggestion
Added the flag `cluster.coreDNS.disabled` to disable coreDNS deployment during the cluster bootstrap.
```

**Review by Unix4ever — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (smira) [reply CHANGELOG.md:32]: I'll get it fixed, but might have slipped already into the docs

Comment (Unix4ever) [reply CHANGELOG.md:32]: Into releases I guess. And maybe we can edit them manually?

**Pull request merged**